README.md

```diff
@@ -1,20 +1,27 @@
 # reactor-downloader
 
-:warning: **This repo is no longer maintained** Due to time constraints, I am no longer maintaining this repository any longer.  I'm archiving it and leaving it read-only so that you can use it as an example of how you might implement something like this on your own.  Feel free to fork and modify it to your needs.
+> [!NOTE]
+> This is a standalone fork of the [@adobe/reactor-downloader](https://github.com/adobe/reactor-downloader) project, which is no longer being maintained. Kudos to @bbythewa and @mikeumus for building the original. Contributions welcome!
 
 Command line tool for downloading a Launch property to a local directory.
 
-## Usage
+## Usage and Installation
+
+Running this tool is currently required to generate your `.reactor-settings.json` file for the Launch Property you want to use in [@adobe/reactor-sync](https://github.com/adobe/reactor-sync).  Before running the downloader tool, you must first have [Node.js](https://nodejs.org/en/) installed on your computer. Your npm version (npm comes bundled with Node.js) will need to be at least 10.15.0. You can check the installed version by running the following command from a command line:
```

Comment:
good add! wish I'd known this the first time I'd tried to use reactor-sync 😂

Reply:
meeeee toooooooo, i think i found it in the depths of an issue report made like... back when DTM was being sunsetted